src/conversations/ConversationsCard.js

```diff
@@ -55,28 +50,33 @@ export default class UserListCard extends Component {
     this.props.initRoutes(getInitialRoutes(this.props.accounts));
   }
 
-  handleFilterChange = (newFilter: string) => {
-    this.setState({
-      filter: newFilter,
-    });
-  }
-
   handleUserNarrow = (email: string) =>
-    this.props.onNarrow(privateNarrow(email));
+    this.props.onNarrow(
+      email.indexOf(',') === -1 ?
+        privateNarrow(email) :
+        groupNarrow(email.split(','))
+    );
+
+  handleSearchPress = () => {
+    const { pushRoute, onNarrow } = this.props;
+    onNarrow();
+    pushRoute('users');
+  }
 
   render() {
-    const { ownEmail, realm, users, presence } = this.props;
-    const { filter } = this.state;
+    const { conversations, realm, users, narrow } = this.props;
 
     return (
       <View tabLabel="People" style={styles.container}>
-        <SearchInput onChange={this.handleFilterChange} />
-        <UserList
-          ownEmail={ownEmail}
-          users={users}
-          presence={presence}
-          filter={filter}
+        <Button
```

Comment:
I don't like have a button called 'Search' here.

If we want to keep private conversations in the right sidebar, we probably want a tab group to switch between private conversations and the list of all users.

Alternatively, we could put the list of private conversations in the left sidebar (like the web app).

@timabbott @borisyankov thoughts?